charts/rasa-x/values.yaml

```diff
@@ -466,7 +466,7 @@ app:
   # name of the custom action server image to use
   name: "rasa/rasa-x-demo"
   # tag refers to the custom action server image tag
-  tag: "0.38.0"
+  tag: "1.1.0a1"
```

Comment:
why here is an alpha version?

Reply:
It's the only image with [1.1.0](https://hub.docker.com/repository/docker/rasa/rasa-x-demo/tags?page=1&ordering=last_updated&name=1.1.0) in Docker Hub. Are we missing a build? 🤔